<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>448</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>448</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
    background-color: #CD96CD;
    width: 10px;
    margin: 0.5px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="message">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplified Arabic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-radius: 10%;
border: 3px ridge #F12FFF;
color: red;
 }</string>
    </property>
    <property name="text">
     <string>مرحبا</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_left">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius: 10%;
border: 3px ridge #F12FFF;
color: red;}

QPushButton::pressed{
border-radius: 10%;
border: 3px ridge #F12FFF;
color: red;
background-color: #F12FFF;
}</string>
    </property>
    <property name="text">
     <string>حسنآ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_right">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius: 10%;
border: 3px ridge #F12FFF;
color: red;}

QPushButton::pressed{
border-radius: 10%;
border: 3px ridge #F12FFF;
color: red;
background-color: #F12FFF;
}</string>
    </property>
    <property name="text">
     <string>موافق</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
